electronic_straw_polls.bs

```diff
@@ -87,15 +87,20 @@ div.newnumbered li:before {
 }
 </style>
 
-# Motivation # {#motivation}
+# Motivation and Summary # {#motivation-summary}
 
 The Standard C++ Committee is starting to conduct more and more of our
-  technical work in a virtual environment (see [[P2145R0]]).
+  technical work in a virtual environment (see [[P2145R0]]), in part due to
+  the COVID19 pandemic.
```

Comment:
Nope you can't say "COVID19". Don't name the virus. I tried to do this once in another document and Jon Kalb and someone else flipped out because they though it was wrong. COVID19 is the disease caused by the virus, not the virus. Something like that. "global pandemic" sounds better. But do we even need to mention it?

Reply:
Updated. yes we n need to say it, because in a few years the context will matter.